pkg/services/ngalert/api/tooling/definitions/contact_points.go

```diff
@@ -85,6 +85,18 @@ type LineIntegration struct {
 	Description *string `json:"description,omitempty" yaml:"description,omitempty" hcl:"description"`
 }
 
+type MqttIntegration struct {
+	DisableResolveMessage *bool `json:"-" yaml:"-" hcl:"disable_resolve_message"`
+
+	BrokerURL          *string `json:"brokerUrl,omitempty" yaml:"brokerUrl,omitempty" hcl:"broker_url"`
```

Comment:
would it be possible to add config for a TLS client cert and key? this was requested for the MQTT data source because some brokers require mTLS

Reply:
Yes, that's definitely something I'd like to do. I planned to do it after this simple MQTT integration is merged